### PR TITLE
Stop Tooltip Truncation

### PR DESCRIPTION
**Pull request by rluodev**
Screenshot 2024-01-26 at 6 08 28 PM

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -396,10 +396,12 @@ summary {
...
+[x-cloak] {
+  display: none !important;
+}
 
 @import 'basscss';
 @import 'space';
 @import 'kbar';
 @import 'utilities';
-@import 'dark';
+@import 'dark';
```
Comment (sampoder) [inline app/assets/stylesheets/application.scss]: let's keep the new line at the end

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -259,7 +259,7 @@ summary {
   border-radius: 0.5rem;
   border: 1px dashed map-get($palette, warning);
   background: rgba(map-get($palette, pending), 0.125);
-  overflow: auto;
+  overflow: visible;
```
Comment (cjdenio) [inline app/assets/stylesheets/application.scss]: This makes `admin_tools` a little wonky on mobile:

https://github.com/hackclub/hcb/assets/34525547/51f6d243-d835-4bc4-9c04-27de6ba69f51

Comment (rluodev) [reply app/assets/stylesheets/application.scss]: huh

Comment (rluodev) [reply app/assets/stylesheets/application.scss]: maybe a css switch on smaller screens?

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (rluodev) [reply app/assets/stylesheets/application.scss]: oh didn't even notice the difference

**Pull request merged**